doc/content/api/node_pipelines.md

```diff
@@ -21,12 +21,21 @@ Run an itk.js Emscripten module with Node.js.
 <dl>
   <dt>`IOTypes.Text`</dt><dd>A UTF8-encoded string.</dd>
   <dt>`IOTypes.Binary`</dt><dd>A binary [`Uint8Array`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Uint8Array).</dd>
-  <dt>`IOTypes.Image`</dt><dd>An [`itk/Image`](./Image.html).</dd>
-  <dt>`IOTypes.Mesh`</dt><dd>An [`itk/Mesh`](./Mesh.html).</dd>
+  <dt>`IOTypes.Image`</dt><dd>An [`itk/Image`](./Image.html). To generate this data type in C++, use the [`itk::ImageFileWriter`](https://itk.org/Doxygen/html/classitk_1_1ImageFileWriter.html), add `BridgeJavaScript` module to the ITK `find_package` `COMPONENTS` in your *CMakeLists.txt*, and use `.json` for the image file name extension.</dd>
+  <dt>`IOTypes.Mesh`</dt><dd>An [`itk/Mesh`](./Mesh.html). To generate this data type in C++, use the [`itk::MeshFileWriter`](https://itk.org/Doxygen/html/classitk_1_1MeshFileWriter.html), add `BridgeJavaScript` module to the ITK `find_package` `COMPONENTS` in your *CMakeLists.txt*, and use `.json` for the image file name extension.</dd>
+  <dt>`IOTypes.vtkPolyData`</dt><dd>A vtk.js [`PolyData`](https://kitware.github.io/vtk-js/docs/structures_PolyData.html). To generate this data type in C++, use the [`vtkJSONDataSetWriter`](https://vtk.org/doc/nightly/html/classvtkJSONDataSetWriter.html), add `vtkIOExport` module to the VTK `find_package` `COMPONENTS` in your *CMakeLists.txt*</dd>
```

Comment:
The vtkJSONDataSetWriter will write polydata as well as imagedata to a vtk.js bundle. I'm not sure to understand what you mean by "to generate that data type use ...".

Reply:
The vtkXMLPolyDataWriter which generate vtp files would be more appropriate?

Reply:
I guess reading further, it seems that the vtk.js format might be indeed a better fit for a format of exchange between Emscripten and plain JS.

Reply:
Yes, that's it: the vtkJSONDataSetWriter format enables efficient  / zero-copy / zero-computation conversions from the Emscripten WebAssembly virtual filesystem.

Similarly, we could add vtk.js vtkImageData support if there is a need for it.

Reply:
ok np.